docs/api/vega-wallet/v2-api/get-started.md

```diff
@@ -40,7 +40,9 @@ See the full set of methods that you can use in the **[Open RPC documentation](.
 The service requires the Origin (or Referrer) HTTP header to be specified in the request. This is usually handled by the browser, so you may not have to do anything, but for software that does not use one of those headers, the request will be rejected.
 :::
 
-Use `POST /api/v2/requests` to communicate with the wallet. The request body is a JSON-RPC 2.0 payload.
+Use `POST /api/v2/requests` to communicate with the wallet. The request body is a JSON-RPC 2.0 payload. 
+
+If you want to use a [long-living token](#generate-a-long-living-token), you'll need to specify it in the `Authorization` HTTP header with the scheme `VWT`. Example: `Authorization: VWT <TOKEN>`
```

Comment:
I forgot to explain how to retrieve this token:
1. For long-living token, as you already know, it can be retrieved from the command-line.
2. For the sessions (`client.connect_wallet` workflow), the token is returned through the `Authorization` HTTP header is the `client.connect_wallet` response. This allows use to use to following technique:
```
response = send_requests("client.connect_wallet")

token = response.Header("Authorization")

send_requests("client.list_keys", header("Authorization", token))
```